extensions/sql-migration/src/constants/strings.ts

```diff
@@ -291,6 +295,9 @@ export function ACCOUNT_ACCESS_ERROR(account: AzureAccount, error: Error) {
 		`${account?.properties?.tenants[0]?.displayName} (${account?.properties?.tenants[0]?.userId})`,
 		error.message);
 }
+export function MI_NOT_READY_ERROR(miName: string, state: string): string {
+	return localize('sql.migration.mi.not.ready', "The Managed Instance '{0}' is currently unavailable for migration because it is currently in the '{1}' state.", miName, state);
```

Comment:
correct wording should be 
"The managed instance..." or  "The Azure SQL Managed Instance..."

Reply:
Please add an actionable step after the message, such as "To continue, select an available managed instance."

Reply:
please double check the correct reference, I expected it to be "SQL Managed Instance" or "Azure SQL Managed Instance" or lower case if referring to it as an asset.

Reply:
Got it, will close with PMs on wording